tools/build-widgets.sh

```diff
@@ -11,5 +11,11 @@ vite build -c=libs/search-widget/vite.config.js -- viewer-widget nuclia-viewer-w
 mv dist/libs/viewer-widget/style.css dist/libs/search-widget/nuclia-viewer-widget.css
 cp dist/libs/viewer-widget/* dist/libs/search-widget
 
+echo "Build chat widget"
+vite build -c=libs/search-widget/vite.config.js -- chat-widget nuclia-chat-widget
+mv dist/libs/chat-widget/style.css dist/libs/search-widget/nuclia-chat-widget.css
```

Comment:
That's correct, but we must also change our `.github/workflows/deploy.yml` so it publishes this file to the CDN.
Visibly we forgot to do so for the viewer style, I guess a command like
```
gsutil copy dist/libs/search-widget/*.css gs://$CDN_STORAGE
```
would fix it for all the existing and future widgets.

Reply:
Fixed